src/neo-vm/ScriptBuilder.cs

```diff
@@ -40,27 +40,27 @@ public ScriptBuilder EmitJump(OpCode op, short offset)
 
         public ScriptBuilder EmitPush(BigInteger number)
         {
-            if (number == -1) return Emit(OpCode.PUSHM1);
-            if (number == 0) return Emit(OpCode.PUSH0);
-            if (number > 0 && number <= 16) return Emit(OpCode.PUSH1 - 1 + (byte)number);
-            return EmitPush(number.ToByteArray());
+            if (number >= -1 && number <= 16) return Emit(OpCode.PUSH0 + (byte)(int)number);
+            byte[] data = number.ToByteArray();
```

Comment:
It's the same, but if someone read the code automatically know the format.

```suggestion
            byte[] data = number.ToByteArray(isUnsigned: false, isBigEndian: false);
```